instrumentation/jaxrs2/src/main/java/brave/jaxrs2/TracingContainerFilter.java

```diff
@@ -46,7 +46,9 @@
   @Context ResourceInfo resourceInfo;
 
   @Override public void filter(ContainerRequestContext request) {
+    if (resourceInfo != null) request.setProperty(ResourceInfo.class.getName(), resourceInfo);
```

Comment:
@llinder you might want to look at this including plumbing as it might be an option for ratpack